frontend/package.json

```diff
@@ -10,10 +10,12 @@
     "fix": "prettier --write . && eslint --fix --ext .jsx ."
   },
   "dependencies": {
+    "axios": "^0.26.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "styled-components": "^5.3.5",
-    "styled-reset": "^4.3.4"
+    "styled-reset": "^4.3.4",
+    "template-fullstack": "file:.."
```

Comment:
Nope ! :warning:

Reply:
Ok, what is the solution it's seems like the styled reset is installed twice ?